js/private/js_run_devserver.mjs

```diff
@@ -430,47 +447,84 @@ async function main(args, sandbox) {
 
     const config = JSON.parse(await fs.promises.readFile(configPath))
 
-    await syncFiles(
```

Comment:
If you [ignore whitespace](https://github.com/aspect-build/rules_js/pull/2230/files?w=1) you can see this `main()` method hardly changed, it's mainly just moving this root `syncFiles()` call into `runIBazelProtocol()`.